scripts/install

```diff
@@ -47,7 +47,10 @@ ynh_clean_setup () {
     ynh_secure_remove "$tmp_dir"
 }
 ynh_setup_source "$tmp_dir"
-mv "$tmp_dir/sources"  "$final_path"
+mv "$tmp_dir/dist"  "$final_path"
```

Comment:
What about the other directories ?
Is `dist` the only interesting directory in this archive ?

Reply:
Dist is the compiled version. Other directories are source code and tools to build the program.